checkov/common/bridgecrew/platform_integration.py

```diff
@@ -1,48 +1,56 @@
+import json
+import logging
+import os
+import re
+import webbrowser
+from json import JSONDecodeError
 from time import sleep
 
 import boto3
 import dpath.util
-import json
-import logging
-import re
 import requests
 import urllib3
-import webbrowser
 from botocore.exceptions import ClientError
 from colorama import Style
-# from git import Repo
-from json import JSONDecodeError
-from os import path
 from termcolor import colored
 from tqdm import trange
 from urllib3.exceptions import HTTPError
 
-from checkov.common.bridgecrew.ci_variables import *
+from checkov.common.bridgecrew import ci_variables
```

Comment:
removed `import *` to be more explicit - this module uses `os` lib, but it gets imported indirectly from `checkov.common.bridgecrew.ci_variables` module. 

All `import *` is bad in general and will have to be addressed anyways should we add a python linting tool, such as `flake8` or `pylint`